src/tests/test_cases.py

```diff
@@ -39,34 +39,34 @@ def test_remove(filled_set):
 
 
 def test_remove_first(filled_set):
-    filled_set.remove(1)
+    assert 0 == filled_set.remove(1)
     assert filled_set[0] == 2
     assert filled_set == OrderedSet(2, 3, 4, 5, 6)
 
 
 def test_remove_last(filled_set):
-    filled_set.remove(6)
+    assert 4 == filled_set.remove(6)
     assert filled_set[-1] == 5
 
 
 def test_add_remove_mixed(filled_set):
-    filled_set.remove(1)
+    assert 0 == filled_set.remove(1)
     assert filled_set[0] == 2
     assert filled_set == OrderedSet(2, 3, 4, 5, 6)
-    filled_set.remove(6)
+    assert 4 == filled_set.remove(6)
     assert filled_set == OrderedSet(2, 3, 4, 5)
-    filled_set.add(7)
+    assert 4 == filled_set.add(7)
     assert filled_set == OrderedSet(2, 3, 4, 5, 7)
 
 
 def test_len(filled_set):
     s = OrderedSet()
-    s.add(1)
-    s.add(1)
-    s.add(1)
-    s.add(2)
-    s.add(3)
-    s.remove(1)
+    assert 0 == s.add(1)
+    assert 0 == s.add(1)
+    assert 0 == s.add(1)
+    assert 1 == s.add(2)
+    assert 2 == s.add(3)
+    assert 0 == s.remove(1)
```

Comment:
We can simplify this passing initial values to the constructor. Like so:
```python3
s = OrderedSet(0, 1, 2)
```

Reply:
Hey, sorry for being this late, I've been really busy lately, and for some reason my bash wasnt recognizing pytest command after pip install, but I already fixed it. I had zero experience with tests, so my code was kinda wrong, but I already made almost all of the fixing that you proposed, except for this one, I dont get whats the purpose of doing s = OrderedSet(0,1,2), and how it can simplify the mentioned code.

Reply:
No problem :)

What I mean is that as you can see, we are adding items one by one which makes this test case more verbose. 
```python
s = OrderedSet()
s.add(1)
s.add(1)
s.add(1)
s.add(2)
s.add(3)
s.remove(1)
```
Without using add and remove statements explicitly, we could have prepared this OrderedSet instance as below instead of the above code snippet:
```python
s = OrderedSet(1, 3)
```
We initialized the OrderedSet instance with initial values of 1 and 3.

This makes the code simpler and cleaner. We just write one line of code instead of seven. Also, there is no need to validate add and remove methods in this test case.

Btw, this is my bad because when I'm writing this test, I should have simplify these lines myself. So, you are fixing my code actually.

Reply:
Got it, I just commited the fixing

Reply:
Thanks, the PR is looking good. Before merging this, we should also update docstring of remove method as well ([this line](https://github.com/buyalsky/ordered-hash-set/blob/e6c09d81849cbb832099792b2411861dd01c687f/src/ordered_hash_set.py#L38)) since it raises ValueError instead of KeyError when removed item is not present.